.github/workflows/build-docs.yaml

```diff
@@ -0,0 +1,22 @@
+name: Docs
+
+on:
+  pull_request: {}
+
+jobs:
+  docs:
+    name: Build docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Fetch latest pctl binary
+        run: |
+          curl -sL https://api.github.com/repos/weaveworks/pctl/releases/latest 2>/dev/null | \
+          jq -r '.assets[] | select(.browser_download_url | contains("linux_amd64")) | .browser_download_url' | \
+          xargs curl -sL | sudo tar xvz -C /usr/local/bin && chmod +x /usr/local/bin/pctl && pctl --help
```

Comment:
While it makes sense, this now adde another connection to pctl from profiles. It was pctl->profiles now it's pctl<->profiles. Guess, what I'm trying to say is, that we are getting closer and closer to just pull pctl into profiles. :)

Reply:
there is also the option to do this:

> move the docs out of the profiles repo? or smth. they should be deployed whenever there are new releases in either pctl or profiles, so whatever makes it easier and more intuitive to do that idk

Reply:
Hmm. `profiles-docs`? That would make sense actually maybe. It would have a dependency on both, profiles and pctl. But at least would remove the dependency between them. ¯\\\_(ツ)\_/¯